From 3ebe52e2a37ff49ffa2811533023dab3d3fbba69 Mon Sep 17 00:00:00 2001
From: Sebastian Bazley <sebb@apache.org>
Date: Tue, 6 Nov 2012 17:31:16 +0000
Subject: [PATCH] IO-356 CharSequenceInputStream#reset() behaves incorrectly in
 case when buffer size is not dividable by data size Extend the test case

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/io/trunk@1406237 13f79535-47bb-0310-9956-ffa450edef68
---
 .../apache/commons/io/input/CharSequenceInputStreamTest.java | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/test/java/org/apache/commons/io/input/CharSequenceInputStreamTest.java b/src/test/java/org/apache/commons/io/input/CharSequenceInputStreamTest.java
index ace5ac2940e..fc7e2b88c32 100644
--- a/src/test/java/org/apache/commons/io/input/CharSequenceInputStreamTest.java
+++ b/src/test/java/org/apache/commons/io/input/CharSequenceInputStreamTest.java
@@ -189,5 +189,10 @@ public void testIO_356() throws Exception {
         testIO_356(10, 10); // OK
         testIO_356(10, 20); // OK
         testIO_356(10, 13); // fails
+        for(int bs = 1; bs <= 10; bs++) {
+            for(int ds = 1; ds <= 20; ds++) {
+                testIO_356(bs, ds);                
+            }
+        }
     }
 }
